app/assets/javascripts/test/tracing-view-screenshot-tests/dataset_rendering.screenshot.js

```diff
@@ -0,0 +1,124 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+const test = require("ava");
+const puppeteer = require("puppeteer");
+const pixelmatch = require("pixelmatch");
+const { PNG } = require("pngjs");
+const fs = require("fs");
+const path = require("path");
+const { screenshotDataset, DEV_AUTH_TOKEN } = require("./dataset_rendering_helpers");
+
+process.on("unhandledRejection", (err, promise) => {
```

Comment:
Shouldn't line 10 - 82 be in another file? Either the helper file you already created or another one?